ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -166,93 +157,112 @@ export const AnacondaWizard = ({ dispatch, isBootIso, osRelease, storageData, lo
+        return stepsOrder.filter(s => !s.isHidden).map(s => {
+            let stepProps = {
+                ariaLabel: `${title} content`,
                 name: s.label,
+                id: s.id,
                 stepNavItemProps: { id: s.id },
+                ...(s.isExpandable && { isExpandable: true }),
                 canJumpTo: canJumpToStep(s.id, currentStepId),
                 isFinishedStep: isFinishedStep(s.id),
-            });
+            };
             if (s.component) {
-                step = ({
-                    ...step,
-                    component: (
+                stepProps = {
+                    ...stepProps,
+                    children: (
                         <s.component
                           idPrefix={s.id}
-                          setIsFormValid={setIsFormValid}
-                          onCritFail={onCritFail}
-                          onAddErrorNotification={onAddErrorNotification}
-                          stepNotification={stepNotification}
-                          isInProgress={isInProgress}
-                          storageEncryption={storageEncryption}
-                          setStorageEncryption={setStorageEncryption}
-                          storageScenarioId={storageScenarioId}
-                          isBootIso={isBootIso}
-                          osRelease={osRelease}
-                          setStorageScenarioId={(scenarioId) => {
-                              window.sessionStorage.setItem("storage-scenario-id", scenarioId);
-                              setStorageScenarioId(scenarioId);
-                          }}
+                          {...componentProps}
                           {...s.data}
                         />
-                    ),
-                });
+                    )
+                };
             } else if (s.steps) {
-                step.steps = createSteps(s.steps);
+                const subSteps = createSteps(s.steps);
+                stepProps = {
+                    ...stepProps,
+                    steps: subSteps
+                };
             }
-            return step;
+            return (
+                <WizardStep key={s.id} {...stepProps} />
+            );
         });
-        return steps;
     };
     const steps = createSteps(stepsOrder);
 
-    const goToStep = (newStep, prevStep) => {
-        if (prevStep.prevId !== newStep.id) {
+    const goToStep = (currentStep, prevStep) => {
+        if (prevStep.prevId !== currentStep.id) {
             // first reset validation state to default
             setIsFormValid(false);
         }
 
         // Reset the applied partitioning when going back from review page
-        if (prevStep.prevId === "installation-review" && newStep.id !== "installation-progress") {
+        if (prevStep.prevId === "installation-review" && currentStep.id !== "installation-progress") {
             setIsInProgress(true);
             resetPartitioning()
```

Comment:
Seems that resetPartitioning is not called when moving from installation-review back. Which corresponds with failing TestBasic.testSidebarNavigation.

Reply:
Ah, you need to replace `prevId` with `id`

Reply:
Ah one more thing probably related also to one of the failing pixel tests: TestStorageMountPoints-testBasic-mount-point-mapping-table. When you go back (either by Back or in sidebar steps, for example to Installation Method) the next steps (for example "Review and Install") are no more disabled (which they used to be).